packages/component-library/src/Sandbox/Sandbox.js

```diff
@@ -105,15 +70,19 @@ const Sandbox = ({
           selectedFoundation={selectedFoundation}
           foundationData={foundationData}
           defaultFoundation={defaultFoundation}
+          allSlides={allSlides}
+          updatePackage={updatePackage}
+          updateFoundation={updateFoundation}
+          foundationMapProps={foundationMapProps}
         />
       </div>
-      <div>
+      <div className={baseMapWrapper}>
         <BaseMap
-          mapboxStyle={mapboxStyle}
+          mapboxStyle={"mapbox://styles/mapbox/dark-v9"}
```

Comment:
I think it makes sense to pull this style into BaseMap so that our different map styles are tracked in one place. Maybe we could make our own "civicMapStyle" prop, with light and dark as options.